src/components/Comments/CommentCard.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react'
+import { deleteComment } from '../../axios';
+import './Comments.css'
+import 'ldrs/dotSpinner'
+
+const spinner = <l-dot-spinner
+    className="comment-spinner"
+    size="30"
+    speed="0.9"
+    color="red"
+></l-dot-spinner>
+
+function CommentCard({ comment, user }) {
+
+    const [commentDeleted, setCommentDeleted] = useState(false);
+    const [failedDelete, setfailedDelete] = useState(false);
+    const [isDeleting, setIsDeleting] = useState(false);
+
+    const unformattedDate = new Date(comment.created_at);
+    const options = {
+        year: "numeric",
+        month: "long",
+        day: "numeric"
+    }
+    const formattedDate = unformattedDate.toLocaleString('en-GB', options);
+
+    function handleClick() {
```

Comment:
I'd probably rename this to handleDelete or something more specific to a deletion!